.gitignore

```diff
@@ -1,7 +1,6 @@
 .DS_STORE
 node_modules/
-temp/*
-!temp/.gitkeep
```

Comment:
Do we not want those two folders to appear in the structure (without files in them) as placeholders?